api/app/Http/Controllers/Api/AuthController.php

```diff
@@ -3,12 +3,16 @@
 namespace App\Http\Controllers\Api;
 
 use App\User;
+use DateTime;
+use App\PasswordReset;
+use Webpatser\Uuid\Uuid;
 use Illuminate\Http\Request;
 use Tymon\JWTAuth\Facades\JWTAuth;
-use App\Http\Controllers\Controller;
+use App\Http\Requests\Api\NewPasswordModel;
+use App\Http\Requests\Api\PasswordResetModel;
 use App\Http\Requests\Api\UserRegistrationModel;
 
-class AuthController extends Controller
+class AuthController extends MailController
```

Comment:
Maybe creating a PasswordResetController? It has nothing to do with Authentication